app/infra/nodeService/nodeService.js

```diff
@@ -21,6 +21,20 @@ class NodeService {
   static hardRefresh() {
     ipcRenderer.send(ipcConsts.HARD_REFRESH);
   }
+
+  static killNodeProcess() {
+    ipcRenderer.send(ipcConsts.QUIT_NODE);
+    return new Promise<string, Error>((resolve: Function, reject: Function) => {
```

Comment:
it is meaningless to wait for response of kill call especially when error is ignored

Reply:
sendSync sends and closes the spacemesh node running, but app is stuck and not closing. await in event listeners file also not working. 
I'm keeping the promise for now and will try to look for another solution.